agent/sidecar_service_test.go

```diff
@@ -2,6 +2,7 @@ package agent
 
 import (
 	"fmt"
+	"github.com/hashicorp/consul/acl"
```

Comment:
Our team uses `goimports -local github.com/hashicorp/consul` to group local packages together. If you use Goland here's the setting: 
![image](https://user-images.githubusercontent.com/30640057/183688085-489c7bef-5fc1-4766-9b8d-96d0cdc5f605.png)

Reply:
done